test/extended/networking/bonds.go

```diff
@@ -0,0 +1,117 @@
+package networking
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	kapiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	nadtypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-network][Feature:bond]", func() {
+	oc := exutil.NewCLI("bond")
+	f := oc.KubeFramework()
+
+	g.It("pod with bond interface should be created ", func() {
+		namespace := f.Namespace.Name
+		podName1, podName2 := "pod1", "pod2"
+		bondnad1, bondnad2 := "bondnad1", "bondnad2"
+
+		g.By("creating network attachment definitions")
+		o.Expect(
+			createNetworkAttachmentDefinition(
+				oc.AdminConfig(),
+				namespace,
+				"macvlannad",
+				`{"cniVersion":"0.4.0","name":"macvlan-nad","plugins":[{ "type": "macvlan"}]}`),
+		).To(
+			o.Succeed(),
+			"unable to create macvlan network-attachment-definition",
+		)
```

Comment:
This doesn't match the code style of the other files

Reply:
ah, someone else complained about line-length. It would be more consistent with other tests to do

    err := long(function, call,
            wrapped, to,
            multiple, lines,
    )
    o.Expect(err).NotTo(HaveOccurred())

Reply:
A previous reviewer asked to:
- inline he function
- be optimistic and use` To(Succeed())` instead of `NotTo(o.HaveOccurred()`

Rolling back those changes.

The `long(..)` is not really that long. Should I oneline them?